### ISSUE TITLE
Build reusable button component to close pop-up modals

### ISSUE THREAD

**Issue by intelagense** [Together-100Devs/Together#334]
## Please provide a summary of the feature
Currently, the close buttons are centered at the top of the modals and says `close` 
Suggest turning the button into a reusable React component moving the `close` button to the right side of the modal and turning it into an `X` icon to match typical expected behavior.

## Are there any potential issues you foresee with this feature
Needs to match the styling in #17 

## Is this feature related to any current or past issues? Please provide the #
This was noticed with PR #333 into issue #303

## Which Branch should this fix be Pulled into?
PR into [development]

*If you're interested in working on this feature, please comment so you can be assigned*

**Comment by colinborows** [comment]
interested, please assign.

**Comment by Caleb-Cohen** [comment]
Hey @colinborows I'm just checking in on this, thank ya!

**Comment by colinborows** [comment]
Hey! After a slight diversion, I am back on the React grind and working on this issue. I have been learning more about contexts and considering how best to implement this. I will reach out to the team with some questions this week.

**Comment by colinborows** [comment]
Update - reached a roadblock with this. Will be teaching out for help tomorrow.

**Comment by Caleb-Cohen** [comment]
> Update - reached a roadblock with this. Will be reaching out for help tomorrow.

Sounds good. Did you get the help you were looking for?

**Comment by luciousmc** [comment]
@colinborows Caleb mentioned you might need some help with this issue. Feel free to reach out to me on discord @brotahman and we can tackle it 😀

**Comment by colinborows** [comment]
Update on this - @luciousmc has been kind enough to help me out with this over the past few days and figure out the best option. Now moving forward creating a reusable button that takes in an icon and a function, rather than simply a close button.

**Comment by Caleb-Cohen** [comment]
> Update on this - @luciousmc has been kind enough to help me out with this over the past few days and figure out the best option. Now moving forward creating a reusable button that takes in an icon and a function, rather than simply a close button.

Anything we can help ya with?

**Comment by emmebravo** [comment]
@intelagense @DevinCLane I think this should be closed. There's a redesign happening, we can bring it up then.
